pkgs/top-level/python-aliases.nix

```diff
@@ -184,6 +184,7 @@ mapAliases ({
   foundationdb60 = throw "foundationdb60 is no longer maintained, use foundationdb71 instead"; # added 2023-06-06
   foundationdb61 = throw "foundationdb61 is no longer maintained, use foundationdb71 instead"; # added 2023-06-06
   functorch = throw "functorch is now part of the torch package and has therefore been removed. See https://github.com/pytorch/functorch/releases/tag/v1.13.0 for more info."; # added 2022-12-01
+  functools32 = throw "functool32 was removed from nixpkgs, because python 3.2 has reach end of life in early 2016"; # added 2024-05-16
```

Comment:
```suggestion
  functools32 = throw "functool32 was removed from nixpkgs, because Python 2.7 has reached end of life in early 2020"; # added 2024-05-16
```

> Backport of the functools module from Python 3.2.3 for use on 2.7 and PyPy.

Reply:
done!